pkgs/development/tools/kubeswitch/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kubeswitch";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "danielfoehrKn";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-IV21VvvjnhTMSYDpRflkqXfbKYPgNeIC/igBoXVwVLo=";
+  };
+
+  vendorSha256 = null;
+
+  subPackages = [ "cmd/main.go" ];
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/danielfoehrkn/kubeswitch/cmd/switcher.version=${version}"
+    "-X github.com/danielfoehrkn/kubeswitch/cmd/switcher.buildDate=1970-01-01"
+
+  ];
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/switch version | grep ${version} > /dev/null
+  '';
```

Comment:
```suggestion
  passthru.tests.version = testers.testVersion {
    package = kubeswitch;
  };
```

Reply:
Doh, I always forget this :P 

Thanks!